go.mod

```diff
@@ -2,9 +2,9 @@ module github.com/prometheus/node_exporter
 
 require (
 	github.com/beevik/ntp v0.2.0
-	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7
+	github.com/coreos/go-systemd v0.0.0-20190204112023-081494f7ee4f
 	github.com/ema/qdisc v0.0.0-20180104102928-b307c22d3ce7
-	github.com/godbus/dbus v4.1.0+incompatible
+	github.com/godbus/dbus v0.0.0-20181101234600-2ff6f7ffd60f
```

Comment:
This seems off, shouldn't it use: https://github.com/godbus/dbus/tree/v5.0.1? Looking at the release log that should be fine.

Reply:
Yea, that should be v5.0.1. Strange.